### PR TITLE
feat: improve typing for open orders and order history

### PR DESCRIPTION
**Pull request by Louis-Amas**

### CODE REVIEW

File: src/queries/orders-history.graphql [hunk 1]
```diff
@@ -84,15 +86,17 @@ query orderHistory(
         gasprice
         gasreq
         gasBase
+        prevGives
+        prevTick
+        tick
+
```
Comment (espendk) [inline src/queries/orders-history.graphql:92]: Is this empty line intentional?
```suggestion
```

File: src/queries/orders-history.graphql [hunk 2]
```diff
@@ -109,6 +113,7 @@ query orderHistory(
           id
           address
         }
+
```
Comment (espendk) [inline src/queries/orders-history.graphql:116]:
```suggestion
```

File: src/transformations.ts [hunk 3]
```diff
@@ -108,7 +124,9 @@ export const getStatus = (lo: LimitOrder): Status => {
     }
   }
 
-  const expiryDate = new Date(lo.expiryDate! * 1000);
+  const expiryDate = lo.expiryDate
+    ? new Date(lo.expiryDate * 1000)
+    : new Date().setTime(Date.now() + 10000);
```
Comment (espendk) [inline src/transformations.ts:129]: Why use this expiry date if there is none?

**Review by espendk — COMMENTED** [review]
Great with stronger typing!
I'm unfamiliar with this code, so a second pair of eyes would be good - maybe @maxencerb ?

File: src/transformations.ts (hunk 3 above)
Comment (Louis-Amas) [reply src/transformations.ts:129]: Night coding. Simplified the code now :)

File: src/types/types.ts [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+  Market,
+  Offer,
+  Order,
+  Sdk,
+  openOrdersQuery,
+  orderHistoryQuery
+} from '../../.graphclient';
 import Big from 'big.js';
-import { BigNumberish } from '@ethersproject/bignumber';
+import { Maybe } from 'graphql/jsutils/Maybe';
```
Comment (maxencerb) [inline src/types/types.ts:13]: Needed ?

**Review by maxencerb — APPROVED** [review]

**Pull request merged**